pom.xml

```diff
@@ -84,6 +84,12 @@
                 <version>2.8</version>
             </plugin>
 
+            <plugin>
```

Comment:
Does this affect the wider maven test - have we checked that the correct number of these are still picked up and run?